# DP: Description: Change default charset for 'russian' locale alias
# DP: Related bugs: #62586
# DP: Dpatch author: Ben Collins
# DP: Patch author: Alistair McKinstry
# DP: Upstream status: Submitted
# DP: Status Details: http://sources.redhat.com/bugzilla/show_bug.cgi?id=120
# DP: Date: 2002-03-10

Index: glibc-2.6/intl/locale.alias
===================================================================
--- glibc-2.6.orig/intl/locale.alias
+++ glibc-2.6/intl/locale.alias
@@ -65,7 +65,7 @@
 polish          pl_PL.ISO-8859-2
 portuguese      pt_PT.ISO-8859-1
 romanian        ro_RO.ISO-8859-2
-russian         ru_RU.ISO-8859-5
+russian         ru_RU.KOI8-R
 slovak          sk_SK.ISO-8859-2
 slovene         sl_SI.ISO-8859-2
 slovenian       sl_SI.ISO-8859-2
